kani-driver/src/args/common.rs

```diff
@@ -42,6 +42,8 @@ pub enum UnstableFeatures {
     CFfi,
     /// Enable concrete playback flow.
     ConcretePlayback,
+    /// Enable function contracts
+    Contracts,
```

Comment:
I don't think you are really using this. Please create an issue to do so and add a TODO.